### DISCUSSION TITLE
Fishnet & Lichess Patron

### DISCUSSION THREAD

**Discussion by jakibbe** [lichess-org/fishnet#209] [Ideas]
A selfish request: Would it be possible to tie in running the Fishnet client (for x amount of time monthly), with Lichess Patron status? I don't always have cash to donate, but my extra laptop is almost always running Fishnet. A profile icon would be a much appreciated perk. I thought that it might be possible in theory since we each run a unique use key tied to our accounts. Thanks for considering! :)

**Comment by ornicar** [comment]
If we do that, then some people will start running fishnet, or pretend to run fishnet, in order to get the Patron status.

I don't think we want to deal with that, it's easier when people just run fishnet for the sake of helping online chess.

**Reply by jakibbe** [reply]
Thanks for your reply,Thibault! It's a privilege and honor!

To your first point, if you have a minimum number of games processed or amount time actively connected to Fishnet (assuming this data exists and is easily accessible), it should be hard to 'game' the system. If more computers are added to Fishnet, regardless of motives, this could take the burden off of Lichess servers.

To your second point, I'm guessing that people donate money to Lichess to get the Patron icon, not only for the sake of helping. If there was an option to toggle off the Patron icon I don't think that many would use it. My point is that we all have our reasons for donating money (and processing power), and many of them are selfish reasons.

I don't play very often on Lichess. I have donated in past solely because my public school students take advantage of the Classroom feature, and I want to support that. I will continue to support Lichess through Fishnet. Thanks!

**Comment by jakibbe** [comment]
@ornicar instead of Patron status, maybe an cool fish icon on a user's profile page that says Fishnet, just as you have icons for streamers, dev team, content team, and moderators. it's a nice token of appreciation, but i don't think anyone will try to 'game the system' to earn it. i was just watching a recent AMA with NoJoke and seeing his profile made me think of it again. thanks for considering and be well! :)

**Reply by niklasf** [reply]
> but i don't think anyone will try to 'game the system' to earn it

I wish, but I have no doubt some people will do crazy things just for a small badge. If anything, it would be more rare/special than the patron wings.

**Reply by jakibbe** [reply]
> If anything, it would be more rare/special than the patron wings.

Exactly! 😁 

Anyway -- all good here! Thanks to you both for responding! 💯
